content/en/docs/ecommerce/shopify/reference/shopify_store/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Shopify stores"
+description: "You need to set up each Shopify shop that you want to integrate with a distinct Shopify store in Business Central."
+lead: ""
+date: 2024-09-13T10:17:12+02:00
+lastmod: 2024-09-13T10:17:12+02:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: ""
+    identifier: "shopify_store-c2a39488e4b4a87853e0a2d025d69897"
+weight: 325
+toc: true
+type: docs
+---
+
+You need to set up each Shopify shop that you want to integrate with a distinct Shopify store in Business Central. You can do so from the **Shopify Integration Setup** administrative section, or you can find the **Shopify Stores** administrative section with the Business Central search function. 
```

Comment:
`You need to set up each Shopify shop that you want to integrate with a distinct Shopify store in Business Central.`
Sounds ambiguous
I suggest: `For each Shopify shop you want to integrate with, you'll need to set up a separate Shopify Store in Business Central.`

Reply:
Thanks, I didn't realize you've assigned yourself here, so I'm sorry for the spam. 
I usually push multiple times since I add reviewers at the end.